common/src/View/ViewPreferencePane.cpp

```diff
@@ -247,7 +259,21 @@ namespace TrenchBroom {
             wxString iconSizes[7] = {"25%", "50%", "100%", "150%", "200%", "250%", "300%"};
             m_textureBrowserIconSizeChoice = new wxChoice(viewBox, wxID_ANY, wxDefaultPosition, wxDefaultSize, 7, iconSizes);
             m_textureBrowserIconSizeChoice->SetToolTip("Sets the icon size in the texture browser.");
+
+
             
+            wxStaticText* fontPrefsHeader = new wxStaticText(viewBox, wxID_ANY, "Fonts");
+            fontPrefsHeader->SetFont(fontPrefsHeader->GetFont().Bold());
+
+            wxIntegerValidator<unsigned int> ValidIntP;
+            wxStaticText* fontPrefsRendererFontSizeLabel = new wxStaticText(viewBox, wxID_ANY, "Renderer Font Size");
+            wxString rendererFontSizes[23] = { "8", "9", "10", "11", "12", "13", "14", "15", "16", "17", "18", "20", "22",
```

Comment:
Not critical but could avoid the 23 constant with
`std::vector<wxString> rendererFontSizes { "8", "9", ..};`

then use rendererFontSizes.data() and rendererFontSizes.size()